### DISCUSSION TITLE
Some feedback about the High gas prices (and fine tune hoarder limits)

### DISCUSSION THREAD

**Discussion by vitt0** [sunflower-land/sunflower-land#2250] [General]
Just like some days ago we had [this discussion](https://github.com/sunflower-land/sunflower-land/discussions/2206) about making BB a priority. It was a great first step in making high gas prices more manageable, but in the last 2 days we had even higher gas spikes (up to 500-800 gwei) with another project that is doing thousands of txs per min and is not enough, so here is some feedback about some things that I think they can be improved about it. 

**- High price for storing progress on chain**

 The gas that we have to pay to store our progress on chain is too expensive since our txs have to update a lot of data, I heard you guys saying in the last dev-talk that you were considering not updating the Bumpkins data when we sync and It could save up to 50% in the gas usage of the txs, but you were uncertantly about losing the ability of track the data in 3th party app's like dune. 

I'm one of the guys that loves to look at this kinda data, but I think that in the current gas conditions is a sacrifice that is worth doing. Other projects don't care so much about having 500-600 gwei because their txs are still worth a few cents, like 0.05-0.10$, but in our case, with this high gwei storing or progress could cost like 1.00-1.50$ in gas, so please consider making this or any type of solution that makes the txs lighter as soon is possible. 🙏
 

**- Add a Discord Bot that show the current gas prices**

![FJ222Lf](https://user-images.githubusercontent.com/11968934/219214839-e0928972-2794-4f56-8b70-ebf85cda514a.png)

  This is a really simple thing to add and it can help to show the daily users that complain in Discord about having to pay too much every time they want to buy BB's or store their progress on chain without even knowing that the gas prices are high. 

**- Hoarder limits**

I know that they are there as a security measure and the contract doesn't allow them to be dynamically according if we buy 1 or 5 BB, also I know than some of them were increased recently, but I still think some of them are pretty low for players that have NFT's that boost their production and use the option of 5 BB's, so please considere raisen them a little more, mainly this ones:
  
-   Carrot: "1000"  (For Green colar is kinda low, maybe raise to 1500-2000)
-   Cauliflower: "1000" (For Golden Cauliflowers + GA is kinda low, maybe raise to 1250)
-   Parsnip: "500" (For Mysterious Parsnip +Parsnip wereable + GA is kinda low, maybe raise to 750-1000)
-   ~~Eggs: "150" (For Chicken Coop is kinda low, maybe raise to 200-250)~~ (already increased to 200)
-   ~~Wood: "600" (For T2+ Beaver + many extra boost is kinda low, maybe raise to 750-1000)~~ (already increased to 1000)
-   Crab: "10" (Not boost related but they are kinda low, maybe raise to 20-25) 

 **And more importantly, change the message when we get the hoarder limit error. It should be explained that we can also expend our resources to be able to continue playing and that "storing progress on chain" is not the only solution. Maybe it could have a button saying "Spend/sell resources" and it takes them to the seed/tools shop according to which resource it is or something like that. 😄** 
 
![hoarder](https://user-images.githubusercontent.com/11968934/219214514-cac00474-99b1-4a7a-b025-846be345b59b.png)

**Comment by Xeko-92** [comment]
I agree. The only hoarder limit I hit is wood, and while the increase from 500 to 600 was nice, it still won't be enough. I don't have the foreman beaver, but I have the apprentice and all the other wood boosts. I will still hit the limit while buying out all the shovels every sync plus enough of the other tools to mine stone, iron, and gold.
